src/litserve/specs/openai_embedding.py

```diff
@@ -152,8 +188,60 @@ def _validate_response(self, response: dict) -> None:
                 f"{EMBEDDING_API_EXAMPLE}"
             )
 
-    async def embeddings(self, request: EmbeddingRequest) -> EmbeddingResponse:
+    def _handle_embedding_response(
+        self, embeddings: Union[List, "np.ndarray", "torch.Tensor", "List[List[float]]"], num_items: int = 1
+    ) -> List[Embedding]:
+        ndim = None
+        if "torch" in sys.modules:
+            import torch
+
+            if isinstance(embeddings, torch.Tensor):
+                ndim = embeddings.ndim
+        if "numpy" in sys.modules:
```

Comment:
```suggestion
        elif "numpy" in sys.modules:
```

Reply:
we want to check both. so it wont be 'else'